README.rst

```diff
@@ -23,50 +23,72 @@ Usage Example
 
 .. code-block:: python
 
-    from adafruit_usb_descriptor import cdc, standard, util
+    # args defines some command-line arguments.
 
-    # langid must always be first
     langid = standard.StringDescriptor("\u0409")
-    manufacturer = standard.StringDescriptor("Manufacturer name")
-    product = standard.StringDescriptor("Product name")
-    # Placeholder for the serial number. C code should dynamically overwrite it.
+    manufacturer = standard.StringDescriptor(args.manufacturer)
+    product = standard.StringDescriptor(args.product)
     serial_number = standard.StringDescriptor("serial number. you should fill in a unique serial number here."[:args.serial_number_length])
     strings = [langid, manufacturer, product, serial_number]
 
+    # vid = 0x239A
+    # pid = 0x8021
+
     device = standard.DeviceDescriptor(
-        idVendor=0x1234,
-        idProduct=0x0001,
+        description="top",
+        idVendor=args.vid,
+        idProduct=args.pid,
         iManufacturer=strings.index(manufacturer),
         iProduct=strings.index(product),
         iSerialNumber=strings.index(serial_number))
 
     # Interface numbers are interface set local and endpoints are interface local
-    # until standard.join_interfaces renumbers them.
+    # until core.join_interfaces renumbers them.
     cdc_interfaces = [
         standard.InterfaceDescriptor(
-            bInterfaceClass=0x2,  # Communications Device Class
-            bInterfaceSubClass=0x02,  # Abstract control model
-            bInterfaceProtocol=0x01,  # Common AT Commands
+            description="CDC comm",
+            bInterfaceClass=cdc.CDC_CLASS_COMM,  # Communications Device Class
+            bInterfaceSubClass=cdc.CDC_SUBCLASS_ACM,  # Abstract control model
+            bInterfaceProtocol=cdc.CDC_PROTOCOL_V25TER,  # Common AT Commands
             subdescriptors=[
-                cdc.Header(bcdCDC=0x0110),
-                cdc.CallManagement(bmCapabilities=0x03, bDataInterface=0x01),
-                cdc.AbstractControlManagement(bmCapabilities=0x02),
-                cdc.Union(bMasterInterface=0x00,
-                          bSlaveInterface=[0x01]),
+                # Working 2.x
```

Comment:
We don't need this in the example. Looks like its a note for yourself.

Reply:
Actually, I think it was a note for yourself :smiley: . That was already there. I've removed the example and just left a link to CircuitPython, so that the example will always be up to date.